src/client/components/Home.js

```diff
@@ -51,10 +51,22 @@ const cards = [
         rules: [
             {
                 text: 'When this creature is the target of a spell or ability, destroy all merfolk.',
-                ability: (scope) => {
-                    let cards = scope.state.cards;
-                    cards = cards.filter(c => c.subtype[0] !== scope.state.Elements.h1[0]);
-                    scope.setState({cards});
+
+                /*
+                 * Probably good to include opponent somehow so that abilities
+                 * can be used on either player
+                 *
+                 * e.g. ability: (player, opponent) => {...}
+                 */
+                ability: (player) => {
```

Comment:
I'm going to have to think about this some more, I don't think player === scope. But this is fine for now